windows-10-setup.md

```diff
@@ -7,45 +10,132 @@
+1. Install some required libraries including zsh, make zsh the default shell, and restart your Windows Terminal
+    - Put this command into your terminal: `sudo apt-get install build-essential python-is-python3 zsh`
+    - Then this command: `chsh -s $(which zsh)`
     - Restart the Windows Terminal
+    - If you get a page full of info about "This is the Z Shell configuration for new users...", press q (Quit and do nothing)
 1. Install oh-my-zsh from inside the Windows Terminal
-    - `sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`
-    - Restart the terminal and open an Ubuntu tab
+    - Enter this command into your terminal (note that it's one long line, even if it displays as two lines on the page where you're reading this): `sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`
+    - Restart the terminal and open an Ubuntu tab with the little plus button in the top left corner
+    - If the prompt in your terminal window is now a little arrow and a tilde (~), instead of "yourname@...", that's OK (you'll change it again in a later step)
 1. Install VS Code if it isn't already installed
+    - https://code.visualstudio.com/download
 1. In your Ubuntu terminal, open VS Code with `code .`
 1. Install the following VS Code extensions
     - ESLint
-    - WSL - Remote
+    - Remote - WSL
     - Live Share (online students only)
     - vscode-icons (optional, but pretty :wink:)
     - Bracket Pair Colorizer (optional)
     - GitLens (optional)
 1. Restart your terminal
 1. Open VS Code again with `code .`
     - This should begin downloading the VS Code Server
-    - When prompted for access, accept it
+    - When prompted for access (by Windows Defender Firewall), click "Allow access"
 1. Install nvm
-    - `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash`
+    - Enter this command into your terminal: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash`
 1. Move the 3 nvm lines from the bottom of `.bashrc` to the bottom of `.zshrc`
+    - This is the trickiest step! 
+    - Run this command to open your .bashrc config file: `code ~/.bashrc`
+    - Scroll down to the bottom of the file and cut the three lines at the bottom that look like this: 
+
+    ```j
+        export NVM_DIR="$HOME/.nvm"
+        [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+        [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+    ```
+
+    - Run this command to open your .zshrc file: `code ~/.zshrc`
+    - Paste the nvm lines you cut from the .bashrc file down at the bottom of the .zshrc file
+1. Get rid of the distracting green highlighting in your terminal (optional, but recommended):
+    - In your .zshrc file (that you opened in the above step), find this line and uncomment it (i.e. delete the `#`):
+    ```
+    # DISABLE LS_COLORS="true"
```

Comment:
There is a better way to disable the green highlighting of folders. It has to do with the file attributes or ownership of the folders. I don't remember the fix off the top of my head, but I can update this section after I grab a student's computer and remind myself. the problem with disabling LS_COLORS is you won't get **any** colours from the `ls` command, which is unnecessarily drastic.

Reply:
Sounds good. I'll leave my sub-optimal tip in for now, and please feel free to replace it once you discover the better way :-)